### ISSUE TITLE
OS X compilation still fails for branches/release-v3.2.0. after v8 – x64

### ISSUE THREAD

**Issue by haf** [EventStore/EventStore#596]
```
	Services/Transport/Http/Controllers/InfoController.cs(48,29): error CS0103: The name `VersionInfo' does not exist in the current context
	ProgramBase.cs(50,39): error CS0103: The name `VersionInfo' does not exist in the current context
	ProgramBase.cs(50,60): error CS0103: The name `VersionInfo' does not exist in the current context
	ProgramBase.cs(50,80): error CS0103: The name `VersionInfo' does not exist in the current context
	ProgramBase.cs(50,101): error CS0103: The name `VersionInfo' does not exist in the current context
	ProgramBase.cs(142,37): error CS0103: The name `VersionInfo' does not exist in the current context
	ProgramBase.cs(142,58): error CS0103: The name `VersionInfo' does not exist in the current context
	ProgramBase.cs(142,78): error CS0103: The name `VersionInfo' does not exist in the current context
	ProgramBase.cs(142,99): error CS0103: The name `VersionInfo' does not exist in the current context

	 0 Warning(s)
	 9 Error(s)
```

Ok, I'm done with this branch now. Going for the 3.0.4 derivative now.

**Comment by gregoryyoung** [comment]
On mac

Commit 5693aad97e1513727cd142516d5feb145b1e8ea0
Merge: 37fa4d1 597d125
Author: James Nugent <james@jen20.com>
Date:   Wed Aug 19 14:14:52 2015 +0100

    Merge pull request #590 from EventStore/remove-own-concurrent

    Remove concurrent collection abstraction for Mono


Build succeeded.
     0 Warning(s)
     0 Error(s)


``` ~/Source/EventStore   release-v3.2.0  mono --version
Mono JIT compiler version 3.12.1 (mono-3.12.0-branch/0849ec7 Wed Jul 29 15:26:19 SAST 2015)

**Comment by haf** [comment]
Ok, let me add more meta-data about my context. One sec.

**Comment by gregoryyoung** [comment]
btw I just merged competing into 3.2.0 we are waiting on CI for testing etc
of merge.

On Fri, Aug 21, 2015 at 3:05 PM, Henrik Feldt <notifications@github.com>
wrote:

> Ok, let me add more meta-data about my context. One sec.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EventStore/EventStore/issues/596#issuecomment-133392759>
> .
>



-- 
Studying for the Turing test
